src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -166,7 +159,7 @@ const AppFrame = ({ appUrl }: Props): ReactElement => {
       messageId: INTERFACE_MESSAGES.ON_SAFE_INFO,
       data: {
         safeAddress: safeAddress as string,
-        network: getNetworkName().toLowerCase() as LowercaseNetworks,
+        network: getChainName().toLowerCase() as LowercaseNetworks,
```

Comment:
@yagopv, please check the `communicator` is correct.

Reply:
Checked the methods in the communicator and now seems to be working fine @iamacook 

All right on my side